code/script.js

```diff
@@ -30,18 +34,114 @@ const showMessage = (message, sender) => {
+  setTimeout(() => selectMovie(year), 1000); 
+
+  inputWrapper.innerHTML = `
+    <button id='1980Btn'>1980s</button>
+    <button id='1990Btn'>1990s</button>
+    <button id='2000Btn'>2000s</button>
+  `
+
+  document.getElementById('1980Btn').addEventListener("click", () => selectMovie('1980s')); // when one of these selections are selected, it leads the chat to the next question which is the selectMovie function 
+  document.getElementById('1990Btn').addEventListener("click", () => selectMovie('1990s'));
+  document.getElementById('2000Btn').addEventListener("click", () => selectMovie('2000s'));
+}
+  
+const selectMovie = (year) => { 
+  inputWrapper.innerHTML ="" // this clears input wrapper from content (the buttons in the previous question)
+  showMessage(`Really feel like watching something from the ${year} `,  'user');
+  showMessage(`${year} you say! Cool, what genre do you feel like?`, 'bot');
+
+  if (year === '1980s') { //fist conditional, different drop down select menus depeneding on users choice of decade
+    inputWrapper.innerHTML = `
+      <select id="select"> 
+        <option value="" selected disabled>Pick a genre</option>
+        <option value="1980's comedy">An 1980's comedy</option>
+        <option value="1980's horror">An 1980's horror</option>
+        <option value="1980's documentary">An 1980's documentary</option>
+      </select>
+    `// this creates a drop-down select menu in the input wrapper 
+  } else if (year === '1990s') {
+      inputWrapper.innerHTML = `
+       <select id="select">
+        <option value="" selected disabled>Pick a genre</option>
+        <option value="1990's comedy">An 1990's comedy</option>
+        <option value="1990's horror">An 1990's horror</option>
+       <option value="1990's documentary">An 1990's documentary</option>
+      </select>
+    `
+  } else {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>Pick a genre</option>
+        <option value="2000's comedy">A 2000's comedy</option>
+        <option value="2000's horror">A 2000's horror</option>
+       <option value="2000's documentary">A 2000's documentary</option>
+      </select>
+    `
+  }
+
+  const select = document.getElementById('select')
+  select.addEventListener('change', () => confirmation(select.value))//this allows user to select from drop down menu leads to next question
+}
+
+const confirmation = (select) => {
+  showMessage(select, 'user');
+  setTimeout(() => showMessage(`Ohh ${select}, fancy! Do you have a Netflix account?`, 'bot'), 1000);
+  
+  inputWrapper.innerHTML = `
+    <button id='yesBtn'>Yes</button>
+    <button id='noBtn'>No</button>
+  `
+
+  document.getElementById('yesBtn').addEventListener("click", () => tooBad('yes'));
+  document.getElementById('noBtn').addEventListener("click", () => tooBad('no'));
+  button.addEventListener('select', () => tooBad(answer));
```

Comment:
This line of code doesn't seem to do anything so it could be removed. `tooBad` is called by the yes or no button in line 118-119